tst/CTA.Rules.Test/CTA.Rules.Test.csproj

```diff
@@ -48,12 +48,10 @@
     <None Update="CTAFiles\project.specific.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="TempRules\microsoft.owin.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Update="TempRules\owin.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="TempRules\" />
```

Comment:
Should we remove `TempRules` here too?

Reply:
Please try running locally in a new repo. When the TempRules folder doesn't exist, it's throwing an error.

Reply:
We are keeping it since it will be needed for future rules files that we are working on to overwrite the S3 rules. Are you saying leaving it empty causes an error or deleting the folder entirely causes an error? I do not have it deleted in this CR and the unit tests are passing currently. Or was this just to answer Shanshan's comment? Your review is marked as need changes Mark.

Reply:
Please check the tests

Reply:
Thanks Shanshan I fixed the code instead to work if folder doesn't exist :-) Nuking the repo did it but i probably just had to delete the debug folder probably would have done it. Thanks.